docs/semgrep-supply-chain/overview.md

```diff
@@ -21,121 +21,112 @@ Object.entries(frontMatter).filter(
 }
 </ul>
 
-# Conceptual overview of Semgrep Supply Chain
-
 <SscIntro />
 
-![Semgrep Supply chain Vulnerabilities page](/img/sc-vulns.png)
+![Semgrep Supply Chain Vulnerabilities page](/img/sc-vulns.png)
 _Figure 1_. Semgrep Supply Chain Vulnerabilities page.
 
-Semgrep Supply Chain parses lock files for a list of dependencies, then scans your codebase using **rules **written with Semgrep's pattern syntax. Supply Chain rules specify the following:
+## Identify open source security vulnerabilities
+
+Semgrep Supply Chain detects [security
+vulnerabilities](https://nvd.nist.gov/vuln/full-listing) in your codebase
+introduced by open-source dependencies, using high-signal rules to determine the
+vulnerability's [reachability](/semgrep-supply-chain/glossary/#reachability).
+
+To do this, Semgrep Supply Chain parses lockfiles for a list of dependencies,
+then scans your codebase using [rules](/running-rules/) that specify the
```

Comment:
```suggestion
then scans your codebase using rules that specify the
```
Running rules is likely going to be folded into other docs, so let's remove the link.